[Fundamentals - Data Products] Fixing broken links (#1952)
cloudflare/cloudflare-docs: products/fundamentals/src/content/data-products/analytics-integrations/elastic/index.md

## Overview

> If you haven’t used Cloudflare Logs before, visit our [Logs documentation](https://developers.cloudflare.com/logs) for more details. Contact your Cloudflare Customer Account Team to enable logs for your account.

Reviewer: ```suggestion
If you have not used Cloudflare Logs before, refer to the [Logs documentation](https://developers.cloudflare.com/logs) for more details. Contact your Cloudflare Customer Account Team to enable logs for your account.
```
